packages/use-contractkit/src/connectors/connectors.ts

```diff
@@ -96,8 +96,7 @@ export class PrivateKeyConnector implements Connector {
 
   async updateFeeCurrency(feeContract: CeloTokenContract): Promise<void> {
     this.feeCurrency = feeContract;
-    const res = await this.kit.setFeeCurrency(feeContract);
-    console.log(res, 'console');
```

Comment:
im supprised you went with not including a check for validity here?

Reply:
This is a mistake I meant to just delete the console.log

Reply:
You wouldn't be able to call this function with something invalid without an error

Reply:
thats what was happening before though. it was called with undefined

Reply:
Sorry I was thinking this was a different spot. Yeah I'll add a verification check.